plugins/modules/ini_file.py

```diff
@@ -176,12 +194,56 @@
     mode: '0600'
     state: present
 
+- name: remove the peer configuration for 10.128.0.11/32
+  community.general.ini_file:
+    path: /etc/wireguard/wg0.conf
+    section: Peer
+    section_has:
```

Comment:
```suggestion
    section_has_values:
```
Same in the examples below.